modules/ROOT/pages/debugging-kernel-crashes.adoc

```diff
@@ -0,0 +1,32 @@
+= Enabling kdump to debug kernel crashes
+
+. Memory must be reserved for the crash kernel during booting of the first kernel. Kernel arguments can be provided like this:
++
+[source, bash]
+----
+sudo rpm-ostree kargs --append='crashkernel=256M'
+----
+xref:kernel-args.adoc[More information] on how to modify kargs via `rpm-ostree`.
+
+. Make sure the path in which the vmcore will be saved is somewhere in the `/var` directory of the FCOS stateroot. It is also possible to write the dump over the network; for additional information, consult the mkdumprd man page and the comments in /etc/kdump.conf.
++
+[source, bash]
+----
+sed -i "s/^path.*/path \/sysroot\/ostree\/deploy\/fedora-coreos\/var\/crash/" /etc/kdump.conf
+----
+
+. Enable the kdump systemd service. 
++
+[source, bash]
+----
+sudo systemctl enable kdump.service
+----
+
+. Reboot your system.
++
+[source, bash]
+----
+sudo systemctl reboot
+----
+
+TIP: For additional information on how to test that kdump is properly armed and how to analyze the dump, refer to the https://fedoraproject.org/wiki/How_to_use_kdump_to_debug_kernel_crashes[kdump documentation for Fedora].
```

Comment:
Should we also link to https://www.kernel.org/doc/html/latest/admin-guide/kdump/kdump.html?